### PR TITLE
Validate DORKL

### PR DESCRIPTION
**Pull request by Pinguim306**
# Validate [DORKL](https://solscan.io/token/BgqjQdnnjRtcELAgkYpfKAxWRqSReWercDdRTH6uLoer)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/DORKLORDonSOL/status/1699814938615713850
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Timoon21** [comment]
Hey, can you add your token on the last line (end of list), not on the middle of the list?
Thanks.

**Comment by Pinguim306** [comment]
@Timoon21 , done!

File: validated-tokens.csv [hunk 1]
```diff
@@ -771,3 +771,4 @@ KIRA,KIRA,kiraZUmSnzgfVfhrdvNj6hxHFaPFTTUk8ioY98cbh6G,9,https://www.arweave.net/
 Phoenix,PHNX,JAzesW3tU2VLDx99pwgkDMca2DTNsEhSWG8sj21B1niz,6,https://bafybeicmwtzskjhcsnoymhsv5fmlruwoxgounmcweiei2qnxh6ayesupiq.ipfs.nftstorage.link,true
 Frog Piss,PISS,7qULVSb7XdoKBDDa7WnuUvoTx5ye4Vrj94iPcwibyQ1F,6,https://shdw-drive.genesysgo.net/BLFjCRyZkaF9GGwiorXPRwYJVpxD7VrrKNH7XccWUCWF/frogpiss.png,true
 OPOS Coin,OPOS,3byG6HEKhG5gUZ5PZU9zKkFaEHYPveGEtvs68LYR8R5N,4,https://arweave.net/fivrynPZdJv5ajd4d1TXW1yHArDA-RvkYYfDoh-_4EU,true
+DORKL LORD,DORKL,BgqjQdnnjRtcELAgkYpfKAxWRqSReWercDdRTH6uLoer,6,https://pbs.twimg.com/profile_images/1699214433820565504/Tih6S4Iz_400x400.jpg,true
```
Comment (Timoon21) [inline validated-tokens.csv:774]:
```suggestion
ᗪOᖇK ᒪOᖇᗪ,DORKL,BgqjQdnnjRtcELAgkYpfKAxWRqSReWercDdRTH6uLoer,6,https://bafkreicufnhs5evfdochjopzpjaeuxoh5s6uhyl54dlfeb2cm4bqjfi4dq.ipfs.nftstorage.link,true
```

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

**Pull request merged**